.travis.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     - stage: Tests
       install:
         - *requirements_all
-        - pip install flake8 flake8-quotes yapf isort
+        - pip install -U flake8 flake8-quotes yapf isort
```

Comment:
what do you think about dumping the test packages versions for both local and CI tests?